components/at_mention/at_mention.tsx

```diff
@@ -59,6 +59,9 @@ export default class AtMention extends React.PureComponent<Props, State> {
     }
 
     handleClick = (e: React.MouseEvent<HTMLAnchorElement>) => {
+        if (e.stopPropagation) {
+            e.stopPropagation();
+        }
```

Comment:
I'm actually confused about why we need this here at all since `e.stopPropagation` should always be defined based on the type definition. Is the type definition perhaps incorrect?

Reply:
this is not part of my pr. But the base react 17 pr.

Reply:
I was confused as well. especially when using the built in React change handler type definitions this is typically very well typed. Could be a remnant of the past though! :D